flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/api/internal/ShowCreateUtilTest.java

```diff
@@ -241,6 +259,46 @@ private static Collection<Arguments> argsForShowCreateTable() {
         return argList;
     }
 
+    private static Collection<Arguments> argsForShowCreateMaterializedTable() {
+        Collection<Arguments> argList = new ArrayList<>();
+        argList.add(
+                Arguments.of(
+                        createResolvedMaterialized(
```

Comment:
nit: maybe have a test for the temporary versions as well. this i a suggestion - tt is up to you if you want to do this.

Reply:
added tests for temorary